documentation/Using-the-SDK/Creating-your-plugin.md

```diff
@@ -4,33 +4,33 @@ This document outlines how to use the Performance Toolkit SDK (SDK) to create
 an SDK plugin. Plugins can be used to expose data from `DataSource`s for
 automation, trace extractors, or viewers (e.g. Windows Performance Analyzer).
 
-Before creating a plugin, it is recommended to read [the overview of the SDK's architecture](../Architecture/Overview.md).
+Before creating a plugin, it is recommended to read [the overview of the SDK's architecture](../Architecture/README.md).
 
 Creating a plugin can be outlined into 4 distinct steps:
```

Comment:
Mind changing this to 4 components or something? If it's "steps" I feel we should have it be a numbered list.

Reply:
I changed it to "actions."